build.gradle

```diff
@@ -52,6 +55,8 @@ dependencies {
     deobfCompile "mezz.jei:jei_1.12.2:4.15.0.281:api"
     runtime "mezz.jei:jei_1.12.2:4.15.0.281"
 
+    deobfCompile "com.azanor:Baubles:1.12-1.5.2"
```

Comment:
👌 I was originally using baubles as a .jar file instead of a maven dependency, so I never remembered to change this.